### PR TITLE
Assorted Tutorials

### PR DESCRIPTION
**Pull request by Gomango999**
Added some tutorials for [complementer](https://igraph.org/python/doc/api/igraph._igraph.GraphBase.html#complementer), [isomorphic](https://igraph.org/python/doc/api/igraph._igraph.GraphBase.html#isomorphic), and [simplify](https://igraph.org/python/doc/api/igraph._igraph.GraphBase.html#simplify). They're all relatively straight forward, so I've combined them into one PR.

### CODE REVIEW

**Comment by ntamas** [comment]
Looks good to me! I'll assign this to @iosonofabio as well so he can merge this if he is okay with the PR.

**Comment by ntamas** [comment]
Also, @iosonofabio ; @Gomango999 has contributed so many good tutorials and examples that I'd really like to get these online as soon as possible. Initially I planned to hold off 0.10.x (i.e. the `develop` branch) until we have finished the integer transition and the API breakage carnage in the C core, but I've started to play around with the idea of releasing `python-igraph` 0.10.0 earlier, based on an interim, non-official version of the C core that has already gone through the integer transition from 32-bit to 64-bit. This is something we should discuss in the next dev meeting.

**Comment by iosonofabio** [comment]
Hehe, sure. One reason I aimed all that doc stuff for develop was to incentivize a new release... Seems like it's working out that way 😉

**Comment by szhorvat** [comment]
There's a long way to go until C/igraph 0.10 will be complete, so python-igraph 0.10 is also quite some time away ...

File: doc/source/tutorials/isomorphism/isomorphism.rst [hunk 1]
```diff
@@ -49,15 +51,13 @@ This example shows how to check `isomorphism <https://en.wikipedia.org/wiki/Grap
...
-The three graphs should now look like this:
-
 .. figure:: ./figures/isomorphism.png
    :alt: A visual representation of three different graphs
    :align: center
 
    Three graphs to be compared for isomorphism
 
-We then have some code to compare the isomoprhism between all three graphs.
+To compare the isomoprhism between all three graphs, we run:
```
Comment (szhorvat) [inline doc/source/tutorials/isomorphism/isomorphism.rst:60]: typo in "isomoprhism"

**Pull request merged**